src/includes/getting-started-config/unity.mdx

```diff
@@ -1,6 +1,19 @@
-You can attach Sentry to a Game Object and initialize it with the DSN programatically:
+Currently, you can only configure Sentry using the Unity tools window.
 
-```csharp
-var sentry = gameObject.AddComponent<SentrySdk>();
-sentry.Dsn = "___PUBLIC_DSN___";
-```
+Open `Tools` -> `Sentry`.
```

Comment:
The message that comes from the common doesn't make much sense here. The config is done in the editor before building/running the game. So this doesn't make much sense:

![image](https://user-images.githubusercontent.com/1633368/115040340-54862c00-9e9f-11eb-96f8-378a30cc52ed.png)

Reply:
I agree, will set it as a `PlatformSection` statement